.changelog/README.md

```diff
@@ -24,13 +24,13 @@ The description could use one of the following two formats:
   ```text
   Remove staking-related roothash messages.
```

Comment:
Below you removed dots from the generated changelog, but both examples here have them (which is also the reason why I always add a dot).

Reply:
Good catch :slightly_smiling_face:!

While assembling Change Logs for the latest releases I've seen most developers did not follow this style (intentionally or unintentionally), so I've decided to suggest to adopt the Git commit message style for Change Log fragments.
It should be familiar to developers and prevent confusion between requiring a Change Log fragment's subject line to end with period and requiring a Git commit message's subject line **not** to end with a period.

Also, the new style is perhaps nicer in the assembled Change Log since subject line are always amended with links to issues/pull requests in parentheses.

I've added the relevant commit describing this change.